src/workflows/dags/satellite_training.py

```
@@ -0,0 +1,75 @@
+
+
+@dag(
+    "AirQo-satellite-model-training-job",
+    schedule="0 0 1 * *",
+    default_args=AirflowUtils.dag_default_configs(),
+    tags=["airqo", "hourly-forecast", "daily-forecast", "training-job", "satellite"],
+)
+def train_satelllite_model():
+    @task()
+    def fetch_historical_satellite_data():
+        from datetime import datetime, timedelta, timezone
```

Ruff

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)